### PR TITLE
Support manually dispatching nightly releases

### PR DESCRIPTION
**Pull request by devongovett**
Not sure if this syntax is correct... YAML is a mystery.

### CODE REVIEW

File: .github/workflows/nightly-release.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: release-nightly
 on:
   schedule:
     - cron: '0 0 * * *'
+  workflow_dispatch: {}
```
Comment (mischnic) [inline .github/workflows/nightly-release.yml:5]:
```suggestion
  workflow_dispatch:
```
If I understand https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration correctly 🤷‍♂️

Comment (DeMoorJasper) [reply .github/workflows/nightly-release.yml]: I think the `{}` is optional but recommended in yaml?

Comment (mischnic) [reply .github/workflows/nightly-release.yml]: I recommend not using yaml at all 😂

**Pull request merged**